x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/retry/RetryingHttpSender.java

```diff
@@ -26,12 +27,16 @@
 import java.net.UnknownHostException;
 import java.util.Objects;
 import java.util.concurrent.Executor;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Supplier;
 
 import static org.elasticsearch.core.Strings.format;
 import static org.elasticsearch.xpack.inference.InferencePlugin.UTILITY_THREAD_POOL_NAME;
 
 public class RetryingHttpSender implements RequestSender {
+
+    static final int MAX_RETIES = 3;
```

Comment:
Another option is we reduce the `RetrySettings::RETRY_TIMEOUT` from 30 to around 5-8 seconds which should achieve something similar.